sdk/openai/Azure.AI.OpenAI/src/Custom/OpenAIClient.cs

```diff
@@ -350,14 +350,18 @@ public OpenAIClient(string openAIApiKey)
             chatCompletionsOptions.InternalNonAzureModelName = _isConfiguredForAzureOpenAI ? null : deploymentOrModelName;
             chatCompletionsOptions.InternalShouldStreamResponse = null;
 
+            string operationPath = chatCompletionsOptions.AzureExtensionsOptions != null
+                ? "extensions/chat/completions"
+                : "chat/completions";
```

Comment:
Nitpick: since we are doing this in multiple places, would it make sense to move this into a method?